templates/prelogin/_sections/software_services/pricing.html

```diff
@@ -1,5 +1,6 @@
 {% load menu_tags %}{% load i18n %}
 <section id="software" class="section section-primary section-software-services-pricing">
+    <span id="software-services-pricing"></span><!-- so /software_services/#software-services-pricing works -->
```

Comment:
Not internally. I added this and "#software-services-bundles" so that the redirects Gillian asked for in [FB 219898](http://manage.dimagi.com/default.asp?219898) work with the full URL and not just the path (see `^software_services/$` 301 redirect in urls.py).